src/ui/main.go

```diff
@@ -36,8 +38,10 @@ func main() {
 	templatesPath := flag.String("templates-path", environment.NewVariable("TEMPLATES_PATH").GetStringValue("templates"), "The UI templates path")
 	configurationPath := flag.String("configuration-path", environment.NewVariable("CONFIGURATION_PATH").GetStringValue("config"), "The configuration files path")
 	logLevel := flag.String("log-level", environment.NewVariable("LOG_LEVEL").GetStringValue("info"), "The log level")
+	delayStartup := flag.Duration("delay-startup", environment.NewVariable("DELAY_STARTUP").GetDurationValue(0), "The delay prior to start up the server. Valid unit times: ms, s, m")
 
 	flag.Parse()
+	time.Sleep(*delayStartup)
```

Comment:
It would be great if we were able to avoid the usage of  `time.Sleep(*delayStartup)`